src/modules/dialog/libkvidialog.h

```diff
@@ -48,6 +49,11 @@ class KviKvsCallbackMessageBox : public QMessageBox, public KviKvsCallbackObject
 	    KviKvsVariantList * pMagicParams,
 	    KviWindow * pWindow, bool modal = false);
 	~KviKvsCallbackMessageBox();
+
+protected:
+	QPushButton * yesButton;
```

Comment:
shouldn't this be m_ ?

Reply:
Fixed in a54359db22a8a65d8018fd76bef5ff5b19ba63f6